### PR TITLE
eject: add page

### PR DESCRIPTION
**Pull request by pxgamer**
Part of #2214

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/eject.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+> Eject cds, floppy disks and tape drives.
+
+- Display the default device:
+
+`eject -d`
+
+- Eject the default device:
+
+`eject`
```
Comment (mfrw) [inline pages/linux/eject.md]: I would suggest swapping this item with the first one.

File: pages/linux/eject.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- Eject a specific device:
+
+`eject {{name_of_device}}`
+
+- Toggle whether a device's tray is open or closed:
+
+`eject {{name_of_device}} -T`
```
Comment (mfrw) [inline pages/linux/eject.md]: Shouldn't the command line options come before the positional parameters?

File: pages/linux/eject.md (hunk 1 above)
Comment (pxgamer) [reply pages/linux/eject.md]: When I was creating this, I actually wasn't sure which way round it should be.

I was going to put it first as it's the call without any arguments.

But, I then also thought, as people may not know what their default drive is, should the check be first in a logical order?

File: pages/linux/eject.md (hunk 2 above)
Comment (pxgamer) [reply pages/linux/eject.md]: It doesn't seem to make any difference, but I'll maybe move them to the start as that's how man pages order them.

File: pages/linux/eject.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+- Display the default device:
+
+`eject -d`
+
+- Eject the default device:
+
+`eject`
+
+- Eject a specific device:
```
Comment (agnivade) [inline pages/linux/eject.md]: In this case, what does it eject ? CD, floppy or tape ? We should mention that.

Comment (pxgamer) [reply pages/linux/eject.md]: It's ordered as:
`cd-rom, scsi, floppy and tape`

Comment (agnivade) [reply pages/linux/eject.md]: Let's mention that in parentheses.

**Comment by agnivade** [comment]
Perhaps also mention a concrete value for {{device}} ?

**Comment by pxgamer** [comment]
@agnivade, sure. Sorry, somehow I missed your comment before.

**Comment by agnivade** [comment]
Umm, sorry if I wasn't clear. I actually meant for all the examples. :P

**Comment by pxgamer** [comment]
Ah, whoops. Should I do different ones for each? Like, `/mnt/cdrom`, `/dev/cdrom`, `/mnt/floppy`

Or just do all of them as `/dev/cdrom`?

**Comment by agnivade** [comment]
/dev/cdrom is fine.

**Pull request merged**